### PR TITLE
Update links for accessibility

### PR DESCRIPTION
**Pull request by microbit-grace**
Underline links where text is inlined and ensure links use brand.600

### CODE REVIEW

File: src/components/AboutDialog.tsx
Comment (microbit-grace) [inline src/components/AboutDialog.tsx:1]: Screenshot 2024-11-13 at 15 59 50

File: src/pages/ImportPage.tsx
Comment (microbit-grace) [inline src/pages/ImportPage.tsx:1]: Screenshot 2024-11-13 at 16 05 40

File: src/components/UnsupportedMicrobitDialog.tsx
Comment (microbit-grace) [inline src/components/UnsupportedMicrobitDialog.tsx:1]: Screenshot 2024-11-13 at 16 05 11

File: src/components/NotFound.tsx
Comment (microbit-grace) [inline src/components/NotFound.tsx:1]: Screenshot 2024-11-13 at 16 04 26

File: src/components/IncompatibleEditorDevice.tsx
Comment (microbit-grace) [inline src/components/IncompatibleEditorDevice.tsx:1]: Screenshot 2024-11-13 at 16 04 00

File: src/components/ErrorHandlerErrorView.tsx
Comment (microbit-grace) [inline src/components/ErrorHandlerErrorView.tsx:1]: Screenshot 2024-11-13 at 16 02 40

File: src/components/ConnectBatteryDialog.tsx
Comment (microbit-grace) [inline src/components/ConnectBatteryDialog.tsx:1]: Screenshot 2024-11-13 at 16 01 56

File: src/components/BrokenFirmwareDialog.tsx
Comment (microbit-grace) [inline src/components/BrokenFirmwareDialog.tsx:1]: Screenshot 2024-11-13 at 16 01 28

File: src/components/IncompatibleEditorDevice.tsx [hunk 1]
```diff
@@ -58,8 +58,9 @@ const IncompatibleEditorDevice = ({
                     values={{
                       link: (children) => (
                         <Link
-                          href="https://support.microbit.org/support/solutions/articles/19000154234-which-version-of-micro-bit-do-i-have-"
                           color="brand.600"
+                          textDecoration="underline"
+                          href="https://support.microbit.org/support/solutions/articles/19000154234-which-version-of-micro-bit-do-i-have-"
```
Comment (microbit-robert) [inline src/components/IncompatibleEditorDevice.tsx:63]: Not related to this PR, but this should presumably also be an external link with the target and rel props.

Comment (microbit-grace) [reply src/components/IncompatibleEditorDevice.tsx:63]: Makes sense [fa86dc1](https://github.com/microbit-foundation/ml-trainer/pull/492/commits/fa86dc143e5a7dea3a63f1e3272979940ae2202b)

**Review by microbit-robert — APPROVED** [review]
LGTM.

**Pull request merged**